adafruit_vs1053.py

```diff
@@ -227,7 +235,7 @@ def play_data(self, data_buffer, start=0, end=None):
         finally:
             self._xdcs.value = True
 
-    def sine_test(self, n, seconds):
+    def sine_test(self, n: int, seconds: int):
```

Comment:
It looks like `seconds` is just being passed to `time.sleep()` so based on the CircuitPython API it can be `float`.